src/btchip_apdu_hash_input_start.c

```diff
@@ -85,7 +85,11 @@ unsigned short btchip_apdu_hash_input_start() {
                 (G_coin_config->kind == COIN_KIND_BITCOIN_CASH ? usingCashAddr
                                                                : 0);
             btchip_context_D.usingOverwinter = 0;
+<<<<<<< HEAD
             if ((G_coin_config->kind == COIN_KIND_ZCASH) || (G_coin_config->kind == COIN_KIND_KOMODO)) {
+=======
+            if ((G_coin_config->kind == COIN_KIND_ZCASH) || (G_coin_config->kind == COIN_KIND_KOMODO) || (G_coin_config->kind == COIN_KIND_ZCLASSIC) || (G_coin_config->kind == COIN_KIND_RESISTANCE)) {
+>>>>>>> upstream/master
```

Comment:
@Kryptoxic ; you have an unaddressed merge here

Reply:
Apologies for the late reply, I was away and busy for the month of December. I have manually looked through the entire project and merged the remaining files.